src/common/store/errors/sagas.ts

```diff
@@ -29,7 +29,7 @@ function* handleError(action: errorActions.HandleErrorAction) {
           console.debug(
             `Additional error info: ${JSON.stringify(action.additionalInfo)}`
           )
-          scope.setExtras(action.additionalInfo)
+          scope.setExtras({ extras: action.additionalInfo })
```

Comment:
Not sure if you need to wrap it in another object - I think it's an empty obj by default. 
Doc: https://getsentry.github.io/sentry-javascript/classes/hub.scope.html#setextras

Reply:
thx sry i made this change b/c of a type issue, but this was the wrong change. fixed.!! ty